sandbox/handlers/error_scenarios/override_contact_details.js

```diff
@@ -176,6 +176,13 @@ export function getAlternateContactDetailsError(
     return [
       400,
       "Client is not allowed to provide alternative contact details",
+      [{
+        code: "CM_CANNOT_SET_CONTACT_DETAILS",
+        title: "Cannot set contact details",
+        field: `${path}/recipient/contactDetails`,
+        message: "Client is not allowed to provide alternative contact details.",
+      },
+    ]
```

Comment:
This indentation looks a little confused? Can you double check?

Reply:
I'll double check that..

Reply:
Be honest, not sure why it was like that, did look odd... Now 2nd times a charm, should be fixed...